src/validation.cpp

```diff
@@ -955,9 +957,10 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state, const C
         pool.RemoveStaged(allConflicting, false);
 
         // This transaction should only count for fee estimation if
-        // the node is not behind and it is not dependent on any other
-        // transactions in the mempool
-        bool validForFeeEstimation = IsCurrentForFeeEstimation() && pool.HasNoInputsOf(tx);
+        // the node is not behind, it is not dependent on any other
+        // transactions in the mempool, and it isn't a BIP 125
+        // replacement transaction (may not be widely supported).
+        bool validForFeeEstimation = IsCurrentForFeeEstimation() && pool.HasNoInputsOf(tx) && !fReplacementTransaction;
```

Comment:
`!fReplacementTransaction` is a lot cheaper of a check than the others, so may as well make it the first one in the group of `&`s. I don't think the optimizer would do that automatically?